@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -220,12 +224,30 @@ export class TileGeometryCreator {
             }
             return filter(technique);
         };
+        this.createTextElements(tile, decodedTile, textFilter);
 
-        this.preparePois(tile, decodedTile);
+        this.createLabelRejectionElements(tile, decodedTile);
+    }
 
-        // TextElements do not get their geometry created by Tile, but are managed on a
-        // higher level.
-        this.createTextElements(tile, decodedTile, textFilter);
+    createLabelRejectionElements(tile: Tile, decodedTile: DecodedTile) {
+        if (decodedTile.pathGeometries === undefined) {
+            return;
+        }
+        const path: THREE.Vector3[] = [];
+        const point = new THREE.Vector3();
+        for (const flatpath of decodedTile.pathGeometries) {
+            for (let i = 0; i < flatpath.path.length; i += 3) {
+                point.set(flatpath.path[i], flatpath.path[i + 1], flatpath.path[i + 2]);
+                // Move to world coordinates, the accuracy isn't important, because we don't need to
+                // worry about z flickering (the path is not rendered). This also makes the code
+                // projecting to screen space in
+                // [[TextElementsRenderer.prepopulateScreenWithBlockingElements]] simpler because
+                // the shift before projection to NDC isn't required.
+                point.add(tile.center);
+                path.push(point);
```

Comment:
Is this working? I think you are missing a `point.clone()`.

Reply:
I think it is fixed in a later commit, it is at least there when I check "Files Changed".